src/clojure/nrepl/cmdline.clj

```diff
@@ -157,7 +157,7 @@
   "Require and resolve `thing`
   `thing` can be a string or a symbol."
   [thing]
-  (let [thing (symbol thing)]
+  (let [thing (edn/read-string thing)]
```

Comment:
Won't this blow up if `thing` is a symbol?

Reply:
Yep, thanks, should be fixed.